ndk/detekt.yml

```diff
@@ -0,0 +1,3 @@
+style:
+  ReturnCount:
+    max: 3
```

Comment:
Was there a particular need for this `detekt` rule?

Reply:
Yeah I took this over from sentry-java, but I think we can actually safely remove this to keep it tight and simple 👍

Reply:
I am not actually against `detekt` or this rule. It's just that, besides the Gradle scripts, we don't have a lot of Kotlin code, and the rule seems very specific. I know it is about how many returns in a function are allowed, but `3` seems like it hit a specific need at some point 😅 .